tests/relay_integration/test_sdk.py

```diff
@@ -127,32 +127,32 @@ def test_bind_organization_context(default_organization):
     }
 
 
+def add_context(scope, organization, **kwargs):
+    scope.set_tag("organization.test", "1")
+
+
 @no_silo_test
-@override_settings(SENTRY_PROJECT=1)
+@override_settings(SENTRY_PROJECT=1, SENTRY_ORGANIZATION_CONTEXT_HELPER=add_context)
 @django_db_all
-def test_bind_organization_context_with_callback(settings, default_organization):
+def test_bind_organization_context_with_callback(default_organization):
     create_default_projects()
     configure_sdk()
 
-    def add_context(scope, organization, **kwargs):
-        scope.set_tag("organization.test", "1")
-
-    settings.SENTRY_ORGANIZATION_CONTEXT_HELPER = add_context
     bind_organization_context(default_organization)
```

Comment:
I think it might read better to leave it as-is (with `def add_context` nested, and only `@override_settings(SENTRY_PROJECT=1)` up top) and then here do a second `override_settings` as

```python
    with override_settings(SENTRY_ORGANIZATION_CONTEXT_HELPER=add_context):
        bind_organization_context(default_organization)
```

And then similarly in `test_bind_organization_context_with_callback_error`.